pkg/sql/sem/builtins/builtins.go

```diff
@@ -7182,12 +7182,7 @@ func recentTimestamp(ctx *tree.EvalContext) (time.Time, error) {
 		telemetry.Inc(sqltelemetry.FollowerReadDisabledCCLCounter)
 		ctx.ClientNoticeSender.BufferClientNotice(
 			ctx.Context,
-			pgnotice.Newf(
-				tree.FollowerReadTimestampFunctionName+
-					" does not returns a value that is less likely to read from the closest replica "+
-					"in a non-CCL distribution, using %s from statement time instead",
```

Comment:
did you still want the "using %s from statement time instead", or some wording according to that effect?

Reply:
I don't, to be honest. I think nobody will understand what that's trying to tell them.
I've considered having the implementation just make the `follower_read_timestamp` function a no-op when there's no license (i.e. don't do a stale read), but I've decided that it's better for the read to be stale indeed for the least divergence from a licensed version.